### PR TITLE
#164689317 Add Image to Intervention Record

### PR DESCRIPTION
**Pull request by ElishaMisoi**
### What does this PR do?
- Adds the ability to add an image to an intervention record

### Description of the task to be completed?
- A user should be able to add images to their intervention records, to support their claims.

### How should you manually test this?

- [x] **Clone the repository**

        $ git clone https://github.com/ann-mukundi/ireporter.git

- [x] **Switch to testing branch**

       $ git checkout ft-add-image-to-intervention-164689317 

- [x] **Switch to the ireporter directory**

       $ cd ireporter/

 

- [x] **Create and activate the virtual environment**

      $  virtualenv -p python .venv

      $ source .venv/bin/activate

- [x] **Install project dependencies**

       $ pip install -r api/requirements.txt

- [x] **Run the database migrations**

       $ python api/manage.py migrate

### Running the application

      $ python api/manage.py runserver

- [x] **Signup and activate user**

           POST <your-localhost>/api/auth/signup/

           POST <your-localhost>/api/auth/activate/

- [x] **Login user**

       POST <your-localhost>/api/auth/login/ 

       Use the token provided after login for Authorization and the endpoint
  
- [x] **Create an intervention record (With and without image provided)**

     

      POST <your-localhost>/api/interventions/ 

- [x] **Update intervention image (With and without image provided)**

       PATCH <your-localhost>/api/interventions/int:intervention-id/

### Testing

      $ python api/manage.py test

### Prerequisites

Python, Git,  Virtualenv

### What are the relevant PT stories?

[#164689317](https://www.pivotaltracker.com/story/show/164689317)

### Screenshots
**Success**
Screen Shot 2019-03-20 at 22 09 29

**Non Existent Record**
Screen Shot 2019-03-20 at 22 10 48

**Missing Field**
Screen Shot 2019-03-20 at 22 11 46

### CODE REVIEW

File: api/config/settings.py [hunk 1]
```diff
@@ -476,3 +478,17 @@
...
+
+# Cloudinary settings for Django. Add to your settings file.
+CLOUDINARY = {
+  'cloud_name': 'ireporter',  
+  'api_key': '774853988846169',  
+  'api_secret': 'C4e4QuEgjubsi-utsCc_yhlgnDQ',  
+}
+
+# Cloudinary settings. Run before pycloudinary is used.
+cloudinary.config(
```
Comment (Benkimeric) [inline api/config/settings.py:495]: I think it's better if this data is kept in the .env file

Comment (ElishaMisoi) [reply api/config/settings.py:495]: Okay, thanks for the note, will make the change.

**Review by Siffersari — APPROVED** [review]
Good job @Elisha-Misoi

**Review by AlvinMugambi — APPROVED** [review]
Good work

**Review by Siffersari — APPROVED** [review]

**Pull request merged**